am-ui/src/assets/i18n/en.json

```diff
@@ -92,6 +93,13 @@
       "email": "E-Mail",
       "phone": "Phone number"
     },
+    "modal": {
+      "title": "Delete profile permanently",
+      "body": "Are you sure you want to delete your profile?"
+    },
+    "deleteProfile": "Delete my profile and all data",

```

Comment:
At the end of some sentences you are using ".", and at another there are no dots. Please use one formatting style.